src/sentry/static/sentry/app/views/projectKeyDetails.jsx

```diff
@@ -24,6 +24,22 @@ import {
 } from '../components/forms';
 import {t, tct} from '../locale';
 
+const getRateLimitError = (obj, key) => {
+  if (!obj || !obj.rateLimit || !Array.isArray(obj.rateLimit)) return null;
+
+  let errors = obj.rateLimit.reduce((acc, errorObj) => {
+    Object.keys(errorObj).forEach(errorObjKey => {
```

Comment:
imo this should be a flatmap and a filter (or a map, and filter, and flatten) but this seems correct as is

Reply:
also why do you iterate over keys if you only care about `key` ? why not just check for that being there?

Reply:
All good points, I've just simplified it even further since we don't care about the error message content at all.

Reply:
nice, I think this makes sense now